lib/travis/build/git.rb

```diff
@@ -27,6 +27,7 @@ def initialize(sh, data)
 
       def checkout
         disable_interactive_auth
+        enable_longpaths
```

Comment:
I would move the condition here: `if config[:os] == 'windows'`, but it's just my personal preference